apps/lend/src/store/createLoanRepaySlice.ts

```diff
@@ -85,18 +85,24 @@ const createLoanRepaySlice = (set: SetState<State>, get: GetState<State>): LoanR
           stateCollateral,
           userCollateral,
           userBorrowed,
-          maxSlippage
+          maxSlippage,
+          userState.debt
         )
         sliceState.setStateByActiveKey('detailInfoLeverage', resp.activeKey, { ...resp.resp, error: resp.error })
-        respError = resp.error
+
+        if (resp.resp && !resp.resp.repayIsAvailable && !resp.resp.repayIsFull) {
+          sliceState.setStateByKey('formStatus', { ...formStatus, error: 'error-full-repayment-required' })
```

Comment:
It would be nice to create a enum with errors and use it, like 
```ts
enum FormError {
  FullRepaymentRequired = 'error-full-repayment-required',
  ....
}
```

```suggestion
          sliceState.setStateByKey('formStatus', { ...formStatus, error: FormError.FullRepaymentRequired })
```